requirements/doc.txt

```diff
@@ -4,28 +4,14 @@
 #
 #    make upgrade
 #
-alabaster==0.7.12         # via sphinx
-babel==2.6.0              # via sphinx
```

Comment:
All the sphinx dependencies disappeared from the file; that doesn't seem right, the `setup.py` on its master branch still lists these.

Reply:
I'm stumped on this. I'm not sure why its removing them

Reply:
Try adding `-v` to that pip-compile command to see if it gives more details?  When I run `make upgrade` in my local Python 3.6.8 environment, it seems to be updating correctly.